overlord/configstate/configcore/corecfg_test.go

```diff
@@ -150,6 +153,15 @@ func (s *configcoreSuite) SetUpTest(c *C) {
 
 	restore := snap.MockSanitizePlugsSlots(func(snapInfo *snap.Info) {})
 	s.AddCleanup(restore)
+
+	// mock an empty cmdline since we check the cmdline to check whether we are
+	// in install mode or uc20 run mode, etc. and we don't want to use the
+	// host's proc/cmdline
+	mockCmdline := filepath.Join(dirs.GlobalRootDir, "cmdline")
+	err := ioutil.WriteFile(mockCmdline, nil, 0644)
```

Comment:
do we need to write the file now that we don't use osutil anymore?

Reply:
yes since boot.ModeAndRecoverySystemFromKernelCommandLine still uses osutil.KernelCommandLineKeyValues anyways